application.go

```diff
@@ -50,6 +50,10 @@ type ApplicationDef struct {
 
 	// e.g. user@host  (no path)
 	Targets map[string]*Target
+
+	// Gives a name to multiple targets, can be used to run multiple camus
+	// commands at once
+	DeployGroups map[string][]string
```

Comment:
WIP PR to discuss proposed config file change.

This keeps the old functionality, and old config files in tact, while allowing us to deploy to multiple/run/set what's active on multiple boxes easily.

An example config file would be

```
{
  "Name": "CloudFunctionApiServer",
  "BuildCmd": "tools/package-for-deploy.sh", 
  "BuildOutputDir": "./deploy",
  "RunCmd": "./prod-run.sh %PORT%",
  "HealthEndpoint": "/camus-health-check",
  "Targets": {
    "uat1": {
      "Ssh": "ubuntu@uat1",
      "Base": 8000 
    },
    "uat2": {
      "Ssh": "ubuntu@uat2",
      "Base": 8000 
    }
  }

  "DeployGroups": {
     "uat": ["uat1", "uat2"]
   }
}
```

This gives us max flexibility, and is seamlessly backward compatible with the 'one box' camus way.